core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -34,6 +34,11 @@ class SearchService(
     with FileChangeListener
     with SLF4JLogging {
 
+  private[indexer] def isUserFile(file: FileName): Boolean = {
+    //may want to cache the mapped results. 
+    (config.allTargets map { vfs.vfile }) exists { file isAncestor _.getName }
```

Comment:
Happy to move away from `{}` block syntax, but can you share a link or example where curly braces fall down? I understand the single vs. multi-line behavior between the two.